src/compiler/diagnosticMessages.json

```diff
@@ -2796,7 +2796,7 @@
         "category": "Message",
         "code": 6132
     },
-    "'{0}' is declared but never used.": {
+    "Property '{0}' is declared but never used.": {

```

Comment:
this is used for all errors. you will need to add a new error message for properties, and use the new error message when reporting errors for properties.